locales/en.json

```diff
@@ -109,5 +109,11 @@
   "Untitled": "Untitled",
   "Warning": "Warning",
   "Warning notice": "Warning notice",
-  "Write something nice…": "Write something nice…"
+  "Write something nice…": "Write something nice…",
+  "Editor width": "Editor width",
+  "Small (default)": "Small (default)",
+  "Medium": "Medium",
+  "Large": "Large",
+  "Extra large": "Extra large",
+  "Full width": "Full width"
```

Comment:
I can see that the existing labels are quasi alphabetically sorted, but some are in a different order. Was this on purpose? I've added my labels at the bottom for now. Please let me know where you want them.

Reply:
This file is not needed, I forgot to remove it. I will do it later.

Reply:
I think a default `en.json` file which contains all labels does have a purpose though. This file can be used to create new locales.

Reply:
The locale files is generated through `yarn build:i18n`.